### DISCUSSION TITLE
科技爱好者周刊（第 248 期）：不要夸大 ChatGPT | ruanyf-weekly

### DISCUSSION THREAD

**Discussion by giscus** [plantree/press-comment#10] [General]
# 科技爱好者周刊（第 248 期）：不要夸大 ChatGPT | ruanyf-weekly

阮一峰的技术周刊

https://ruanyf-weekly.vercel.app/weekly/issue-248

**Comment by terminalqo** [comment]
这阅读体验真的好太多了

**Reply by hjg66-5** [reply]
确实

**Comment by hjg66-5** [comment]
我在站长的仓库issues了，希望能被看到
https://github.com/plantree/ruanyf-weekly/issues/5
